### ISSUE TITLE
No documentation generated when quotes in pathname

### ISSUE THREAD

**Issue by hasanabuzayed** [realm/jazzy#1049]
I've being trying run jazzy on different projects on my machine but it doesn't seems to be working, I tried uninstalling and reinstalling it, yet the same results, updated xcode-select, gem, ruby.
these are different ways I tried to run jazzy on my project: 
1. just simply `jazzy` command
2. `jazzy --min-acl internal/private`
3. file .json/yaml
```
{
    "xcodebuild_arguments": ["-project", "MyProject.xcodeproj", "-configuration", "Debug", "-target", "MyProject"],
    "author": "Company Inc",
    "author_url": "https://github.com/project/MyProject",
    "module": "MyProject",
    "sdk": "iphonesimulator",
}
```

as for the output I always get
```
Running xcodebuild
0% documentation coverage with 0 undocumented symbols
building site
building search index
jam out ♪♫ to your fresh new docs in `docs`
```

This is my system:
**macOS**: `10.14.2 (18C54)`
**Xcode**: `10.1 (10B61)`
**Swift**: `4.2.1`
**Jazzy**: `0.9.5`
**Ruby**: `ruby 2.3.7p456 (2018-03-28 revision 63024) [universal.x86_64-darwin18]`

**Comment by johnfairh** [comment]
This means SourceKitten has identified a Swift module `MyProject` being built but can't find any Swift source files that are part of it.

You'll need to look at the output from, looking at the last of your examples above, `xcodebuild clean build -project MyProject.xcodeproj -configuration Debug -target MyProject` unless you can share it.

The code that tries to parse the output is [here](https://github.com/jpsim/SourceKitten/blob/4be914be6fa49cd30b1e7ef5d32d06c037d8f469/Source/SourceKittenFramework/Xcode.swift#L115) and then [here](https://github.com/jpsim/SourceKitten/blob/4be914be6fa49cd30b1e7ef5d32d06c037d8f469/Source/SourceKittenFramework/Module.swift#L101#L119) -- that final filter is evaluating to the empty list.

My blind guess is either an Xcode configuration issue or some unusual filesystem characters somehow confusing a regexp.  Jazzy is a bit more downlevel on SourceKitten than I'd thought but it should be fine.

**Comment by hasanabuzayed** [comment]
You blind guess was the wildest, my colleague thought of so many different scenarios but folder naming issue was not one of them.
So this `/Users/username/Development/Xcode'sProjects/Project/MyProject` was my `pwd` when I faced the issue, but then I tried moving the project to a different directory when no `'` character, and it worked. 
I would usually face such issues if I have spaces on my folders names which I stopped don't but I had no idea that `'` would cause such issues as well.

Thank you so much @johnfairh I spent

**Comment by jpsim** [comment]
https://github.com/jpsim/SourceKitten/pull/577 was merged

**Comment by fbeeper** [comment]
I just had a similar issue with parenthesis in my project path. Jazzy would fail to generate my documentation while raising a bunch of `WARNING: No documented top-level declarations match name XXX specified in categories file`. But it works correctly when I remove the parenthesis. 

As far as I understand, jpsim/SourceKitten#577 is a fix for escaping and this would be covered as well. However, I thought it may be useful to add this information here for anyone lost with similar warnings not noticing the problem can be rooted on the path characters.

Thanks for your great work :)

**Comment by johnfairh** [comment]
Included in jazzy master now via #1076
